### PR TITLE
Feature/sof 5318

### PR DESCRIPTION
**Pull request by dmitry-spodarets**

### CODE REVIEW

File: .github/workflows/update.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+      - name: Checkout LFS objects
+        run: git lfs pull
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 'pypy-2.7'
+
+      - name: Build pages
+        uses: dmitry-spodarets/mkdocs-s3-publish-action@main
```
Comment (timurbazhirov) [inline .github/workflows/update.yml:24]: Should not be using personal account references

File: .github/workflows/update.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Checkout LFS objects
+        run: git lfs pull
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 'pypy-2.7'
```
Comment (timurbazhirov) [inline .github/workflows/update.yml:21]: Should switch to py3

File: .github/workflows/update.yml (hunk 1 above)
Comment (dmitry-spodarets) [reply .github/workflows/update.yml]: fixed

File: .github/workflows/update.yml (hunk 2 above)
Comment (dmitry-spodarets) [reply .github/workflows/update.yml:21]: @AcylSilane said that at the moment we can't go to py3.  We need to update one of the libraries that only work with py2. 
We need to create a separate task about migrating the documentation to py3 and updating all the packages to the latest versions.

Comment (tjduigna) [reply .github/workflows/update.yml:21]: Why are we using pypy at all?

Comment (timurbazhirov) [reply .github/workflows/update.yml:21]: Which library is that?

I was able to run the docs with py3 a while ago, it shouldn't be all that difficult, but yes - let's have a separate task to track it.

File: extra/js/ga.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+gtag('js', new Date());
+gtag('config', 'UA-69270713-5');
```
[the remark this answers is not in the document]
Comment (dmitry-spodarets) [reply extra/js/ga.js:4]: New. I created a separate tracker in GA for each resource to not overlap.

File: .github/workflows/update.yml (hunk 2 above)
Comment (dmitry-spodarets) [reply .github/workflows/update.yml:21]: https://exabyte.slack.com/archives/CFGN3K05C/p1638552607082400?thread_ts=1638551239.080400&cid=CFGN3K05C

I will create the task to update all packages and migrate the environment to py3

**Comment by dmitry-spodarets** [comment]
I close this PR, and create a new one, which will include all the changes associated with https://exabyte.atlassian.net/browse/SOF-5426

**Pull request closed without merging**